cristin-commons/src/main/java/no/unit/nva/utils/AccessUtils.java

```diff
@@ -63,4 +82,45 @@ private static boolean requesterHasNoAccessRightToUseNationalIdentificationNumbe
     private static boolean requesterHasNoAccessRightToEditProjects(RequestInfo requestInfo) {
         return !requestInfo.userIsAuthorized(EDIT_OWN_INSTITUTION_PROJECTS);
     }
+
+    public static String getBackendAccessToken() throws IOException, InterruptedException {
+        var cognitoTokenUrl = URI.create("https://nva-dev.auth.eu-west-1.amazoncognito.com/oauth2/token");
```

Comment:
This URI is hardcoded to dev, eu, etc. Will this fix beyond (also outside this string) work across envs?